### ISSUE TITLE
Memory Check goes on CRITICAL if statuses are None on Dell PowerEdge R6615

### ISSUE THREAD

**Issue by ghost** [bb-Ricardo/check_redfish#184]
Hello everyone,
while executing the following check:

```bash
./check_redfish.py --memory -H X.X.X.X -f <REDECTED> -t 60 --detailed
```

we see the following output:

```bash
[CRITICAL]: Memory Summary health is reported as 'UNKNOWN' but all 4 memory modules (Total 128.0GB) are in good condition
[OK]: Memory module DIMM A3 (32.0GB) status is: None
[OK]: Memory module DIMM A1 (32.0GB) status is: None
[OK]: Memory module DIMM A4 (32.0GB) status is: None
[OK]: Memory module DIMM A2 (32.0GB) status is: None
```

The script prints `[OK]: Memory module XXX (YGB) status is: None` which returns `[OK]` but then returns a `CRITICAL` state in the output even if the memory summary health is reported as `UNKNOWN`, probably because the statuses are `None`.

Attached to this issue you can find the full script dump in verbose mode, where you can see that for each memory module, the retrieved values for `Status.Health` and `Status.State` are always `None`.

How can we possibly solve this?
Thank you.

[redfish-memory-issue.txt](https://github.com/user-attachments/files/23257097/redfish-memory-issue.txt)

**Comment by HHerrgesell** [comment]
Seems strange to me, are you able to see on the iDRAC Web Console a health/state of the individual memory modules?

**Comment by bb-Ricardo** [comment]
Hi,

this seems to be about bug in the iDRAC. have you tried restarting it or updating to a newer version?

**Comment by ghost** [comment]
Hi,
I will check and let you now, thank you.

**Comment by bb-Ricardo** [comment]
Closing this issue due to no response. Feel free to reopen.
